ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1250,6 +1249,25 @@ def test_ipahealthcheck_verify_perms_for_source_files(self,
         assert data[0]["kw"]["type"] == "mode"
         assert data[0]["kw"]["expected"] == "0600"
 
+    def test_ipahealthcheck_verify_perms_upgrade_log_file(self,
```

Comment:
Except for the "touch" command and the file path, this test is identical to `test_ipahealthcheck_verify_perms_for_source_files`. Maybe worth refactoring the code?